test/regression/cases/process_agent_standard_check_with_stats/experiment.yaml

```diff
@@ -0,0 +1,16 @@
+optimization_goal: egress_throughput
+erratic: true
+
+environment:
+  DD_TELEMETRY_ENABLED: true
+  DD_PROCESS_CONFIG_PROCESS_DD_URL: http://127.0.0.1:9092
+  # For regression detection we only care about the processes generated inside the container
+  # so this disables checking of the processes of the host the container is running on
+  HOST_PROC: /proc
+  DD_SYSTEM_PROBE_PROCESS_ENABLED: true
+
+profiling_environment:
```

Comment:
Could you copy in the system probe profiling config from [here](https://github.com/DataDog/single-machine-performance/blob/2a0b13fb600ae9ba3d52b0b4067058af16225477/experiments/regression/agent/cases/file_tree_demo/experiment.yaml#L20-L31)?